### ISSUE TITLE
Remove "no results" warning

### ISSUE THREAD

**Issue by ii14** [folke/trouble.nvim#83]
Hey, I don't see any way to silence the `[Trouble] no results` message. Should this really be a warning anyway?

**Comment by jnalley** [comment]
I created a PR #89 that prevents the message when auto close is enabled.  I'm not sure if that's sufficient in all cases, but that's where I found the message to be distracting.

**Comment by ii14** [comment]
I feel that this message should appear only (if ever) when you're opening the window for the first time, kinda like when quickfix throws an error when there's no items. No errors in your code should not be a warning :) I don't use the auto close option, so that's not going to work for me.

**Comment by jnalley** [comment]
>No errors in your code should not be a warning :)
Yeah, I agree.  Perhaps the warning could just be removed entirely.

**Comment by wookayin** [comment]
We still don't have this feature. Can I completely remove this warning message? For example, the warning message will show when navigating to another buffer who has no results.

**Comment by wookayin** [comment]
Requested in #100
